tools/pkglint/package.json

```diff
@@ -16,7 +16,7 @@
     "pkglint": "bin/pkglint"
   },
   "scripts": {
-    "build": "tsc -b && tslint -p . && chmod +x bin/pkglint",
+    "build": "tsc -b && tslint -p . && chmod +x bin/pkglint.js",
```

Comment:
why?

Reply:
Ohh this can be reverted... I was confused by a weird behavior of yarn that I thought was caused by the "bin" directive, but was due to it being installed by file-reference instead of via repository-peer-linking....